<!DOCTYPE html><html><head><title>R: SNP LD weights from the phenocause example dataset</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.css">
<script type="text/javascript">
const macros = { "\\R": "\\textsf{R}", "\\code": "\\texttt"};
function processMathHTML() {
    var l = document.getElementsByClassName('reqn');
    for (let e of l) { katex.render(e.textContent, e, { throwOnError: false, macros }); }
    return;
}</script>
<script defer src="https://cdn.jsdelivr.net/npm/katex@0.15.3/dist/katex.min.js"
    onload="processMathHTML();"></script>
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body><div class="container"><main>

<table style="width: 100%;"><tr><td>phenocause.ldak_weights</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>SNP LD weights from the phenocause example dataset</h2>

<h3>Description</h3>

<p>A list of data frames containing LDAK-computed Linkage Disequilibrium (LD) weights for all SNPs in the example genetic dataset.
</p>


<h3>Usage</h3>

<pre><code class='language-R'>phenocause.ldak_weights
</code></pre>


<h3>Format</h3>

<p>A list of three <code>data.frame</code> objects with two columns, corresponding to the *weights.short files generated by ldak <code>--calc-weights-all</code>:
</p>

<dl>
<dt><code>rsid</code></dt><dd><p>The SNP identifier.</p>
</dd>
<dt><code>ldak_weight</code></dt><dd><p>The LDAK-computed LD weight.</p>
</dd>
</dl>



<h3>Details</h3>

<p>An LDAK weight for a given SNP measures the amount of local LD between that SNP and its neighbors. Within <code>phenocause</code>, these weights can be used to define sampling probabilities for causal sites, allowing for simulations where causal variants are enriched in either high- or low-LD regions.
</p>


<h3>See Also</h3>

<p><code>sample_causal_sites</code>, <code>metadata</code>
</p>


<h3>Examples</h3>

<pre><code class='language-R'># Load the weights data
data(phenocause.ldak_weights)
# View chromosome names
names(phenocause.ldak_weights)
# View weights for chromosome 1
head(phenocause.ldak_weights[["chr1"]])

</code></pre>

</main>

</div>
</body></html>
